thunder/torch/__init__.py

```diff
@@ -1867,6 +1867,31 @@ def relu6(a: TensorProxy, /, inplace: bool = False) -> TensorLike:
 _inplace_to_out_of_place[relu6] = relu6, 1
 
 
+@torchsymbol(torch.rrelu, torch.nn.functional.rrelu, id="torch.rrelu", is_method=False)
+def rrelu(
+    a: TensorProxy, /, lower: float = 0.125, upper: float = 1.0 / 3, training: bool = False, inplace: bool = False
```

Comment:
It's funny to see a constant like `1.0 / 3` as a default. It is completely correct to do this, and it will only be computed once when the function is defined, it's just an unusual pattern.